core/java/iesi-core/src/main/java/io/metadew/iesi/connection/database/h2/H2DatabaseConnectionService.java

```diff
@@ -32,6 +32,11 @@ public String getDriver(H2DatabaseConnection databaseConnection) {
         return H2DatabaseConnection.class;
     }
 
+    @Override
+    public String refactorLimitAndOffset(H2DatabaseConnection databaseConnection, String query) {
+        return query;
+    }
+
```

Comment:
All XDatabaseConnectionService's extend either DatabaseConnectionService or SchemaDatabaseConnectionService (which extends DatabaseConnectionService). You could implement the default behavior in the DatabaseConnectionService class